Formula/gold.rb

```diff
@@ -0,0 +1,43 @@
+class Gold < Formula
+  desc "GNU gold linker"
+  homepage "https://www.gnu.org/software/binutils/binutils.html"
+  url "https://ftp.gnu.org/gnu/binutils/binutils-2.38.tar.xz"
+  mirror "https://ftpmirror.gnu.org/binutils/binutils-2.38.tar.xz"
+  sha256 "e316477a914f567eccc34d5d29785b8b0f5a10208d36bbacedcc39048ecfe024"
+  license all_of: ["GPL-2.0-or-later", "GPL-3.0-or-later", "LGPL-2.0-or-later", "LGPL-3.0-only"]
+
+  depends_on :linux
+
+  uses_from_macos "zlib"
+
+  def install
+    args = [
+      "--disable-debug",
+      "--disable-dependency-tracking",
+      "--enable-deterministic-archives",
+      "--prefix=#{prefix}",
+      "--infodir=#{info}",
+      "--mandir=#{man}",
+      "--disable-werror",
+      "--enable-gold",
+      "--enable-interwork",
+      "--enable-multilib",
+      "--enable-plugins",
+      "--enable-targets=all",
+      "--with-system-zlib",
+      "--disable-nls",
+    ]
+
+    system "./configure", *args
+    # Pass MAKEINFO=true to disable generation of HTML documentation.
+    # This avoids a build time dependency on texinfo.
+    system "make", "all-gold", "MAKEINFO=true"
+    system "make", "install-gold", "MAKEINFO=true"
+    bin.install_symlink "ld.gold" => "gold"
+    system "strip", *Dir[bin/"*", lib/"*.a"]
```

Comment:
```suggestion
    system "strip", *bin.children, *lib.children
```

Reply:
I’ll also fix this in `binutils` because this is just copied from that formula anyway.

Reply:
It looks like we need to restrict `lib.children` to only files that end in `.a`, because there are some `.la` files that fail when we try to strip them. Is there a way to restrict to `*lib.children` to `*.a` files or do we have to use `*Dir[lib/"*.a"]`?

Reply:
Try
```ruby
lib.glob("*.a")
```
It does the same as using `Dir`, but `Dir` is designed for operating over directories so `Pathname#glob` is semantically better.

Reply:
Thanks, that does look better!